subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_pdu.h

```diff
@@ -65,6 +65,26 @@ static inline struct pdu_adv *lll_adv_scan_rsp_peek(struct lll_adv *lll)
 	return (void *)lll->scan_rsp.pdu[lll->scan_rsp.last];
 }
 
+static inline struct pdu_adv *lll_adv_pdu_latest_peek(struct lll_adv_pdu *pdu)
+{
+	uint8_t first;
+
+	first = pdu->first;
+	if (first != pdu->last) {
+		first += 1U;
+		if (first == DOUBLE_BUFFER_SIZE) {
+			first = 0U;
+		}
+	}
+
+	return (void *)pdu->pdu[first];
+}
```

Comment:
It's a bit confusing that the function peeks the `latest` and then we use a variable named `first` to return the `latest`.

Reply:
> first is the first valid (and currently used) PDU in the double buffer,

If that is the way you describe it, `first_valid` might have been a better choice of a name :) 

It isn't something that needs to be changed, but it is a interesting (and unconventional) naming